static-analysis/action.yaml

```diff
@@ -11,12 +11,17 @@ inputs:
 runs:
   using: composite
   steps:
-    - run: echo "pausing checks"
+    - name: Checkout
+      if: ${{inputs.checkout-repo == 'true'}}
+      uses: actions/checkout@v3
```

Comment:
We can use v4

Reply:
Looking at the new GHA api it seems we could use the `container` keyword

https://semgrep.dev/docs/semgrep-ci/sample-ci-configs/#sample-github-actions-configuration-file

Reply:
> Looking at the new GHA api it seems we could use the `container` keyword
> 
> https://semgrep.dev/docs/semgrep-ci/sample-ci-configs/#sample-github-actions-configuration-file

@tagoro9 I was wondering what the change might look like since composite actions do not directly support the container field ?

Reply:
I see, I read the example too quickly and for this action we might not be able to use it...